docs/modules/ROOT/pages/reference/extensions/rest-openapi.adoc

```diff
@@ -44,3 +44,26 @@ Or add the coordinates to your existing project:
 ifeval::[{doc-show-user-guide-link} == true]
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
 endif::[]
+
+[id="extensions-rest-openapi-usage"]
+== Usage
+[id="extensions-usage-required-dependencies"]
+=== Required Dependencies
+
+A `RestProducerFactory` implementation must be available when using the rest-openapi extension. The currently known extensions are:
+
+* camel-quarkus-http
+* camel-quarkus-netty-http
+
+Maven users will need to add the following dependency to their `pom.xml` for this component:
```

Comment:
Could some users think that the dependency to http is to always be added ?

Maybe:

```
One this two extensions must be added manually. For instance, a maven users could add :
```

Reply:
Great, looks good :+1: